### PR TITLE
align widget ID's according to name

### PR DESCRIPTION
**Pull request by meirwah**

### CODE REVIEW

**Comment by aviadl** [comment]
@meirwah i think that you need to fix them on dashboards as well

**Comment by meirwah** [comment]
You asked for it!!!

File: Widgets/widget-ActiveIncidentsBySeverity.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "id": "6b25afb6-7a95-4878-a24c-24cf067eb38a",
+  "id": "active-incidents-by-severity",
```
Comment (dorsha) [inline Widgets/widget-ActiveIncidentsBySeverity.json]: The dashboard is still using the old ID.

image

File: Widgets/widget-UnassignedIncidentsNumber.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "id": "63de68a7-1da4-489e-8939-b8566da5fc58",
+  "id": "unassigned-incidents",
```
Comment (orius123) [inline Widgets/widget-UnassignedIncidentsNumber.json]: should add `-number`

File: Widgets/widget-ActiveIncidentsBySeverity.json (hunk 1 above)
Comment (orius123) [reply Widgets/widget-ActiveIncidentsBySeverity.json]: The widgets in the dashboard are copies of the widgets when we created them, they don't update if we update the widgets.

**Pull request merged**